src/app/routers/field.py

```diff
@@ -0,0 +1,79 @@
+"""
+Field router.
+"""
+from flask import request, jsonify
+from flask_restx import fields, Resource
+from flask_login import current_user
+from app import API
+from app.services.field_crud import FieldOperation, FieldService
+
+FIELDS_NS = API.namespace('fields', description='NgFg APIs')
+
+FIELD_MODEL = API.model('Field', {
+    'name': fields.String(required=True),
+    'owner_id': fields.Integer(required=True),
+    'field_type': fields.Integer(required=True)
+})
+AUTOCOMPLETE_MODEL = API.model('Setting_autocomplete', {
+    "data_url": fields.String,
+    "sheet": fields.String,
+    "from_row": fields.String,
+    "to_row": fields.String,
+})
+RANGE_MODEL = API.model('Range', {
+    'min': fields.Integer,
+    'max': fields.Integer
+})
+
+EXTENDED_MODEL = API.inherit('Extended_field', FIELD_MODEL, {
+    "range": fields.Nested(RANGE_MODEL),
+    "choice_options": fields.List(fields.String),
+    "setting_autocomplete": fields.Nested(AUTOCOMPLETE_MODEL)
+})
+
+
+@FIELDS_NS.route("")
+class FieldAPI(Resource):
+    """
+    Field API
+    """
+
+    @API.expect(EXTENDED_MODEL)
+    # pylint: disable=no-self-use
+    def post(self):
+        """
+        Field POST method
+
+        :return: json
+        """
+        req = request.json
+        data = FieldOperation.create(**req)
+
+        return jsonify(data)
```

Comment:
Validation?
- if create() returns None
- if request body is incorrect
- onwer_id in request body cannot differ from request user id

Reply:
fixed